crates/contracts/solidity/Trader.sol

```diff
@@ -91,7 +91,8 @@ contract Trader {
                 // revert. Instead, we fall-through so that we handle insufficient sell
                 // token balances uniformly for all tokens.
                 if (address(this).balance >= amountToWrap) {
-                    INativeERC20(nativeToken).deposit{value: amountToWrap}();
+                    try INativeERC20(nativeToken).deposit{value: amountToWrap}() {}
+                    catch {}
```

Comment:
nit: Deposit (given the contracts you use) can never revert, so I don't think `try/catch` wrapping is needed).